packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -66,12 +67,22 @@
 		"react-dom": "^17.0.1"
 	},
 	"devDependencies": {
+		"@fluid-example/example-utils": "workspace:~",
```

Comment:
I don't see usages for example-utils, aqueduct, container-runtime-definitions, core-interfaces, and runtime-utils. Are those necessary here?

Reply:
@chentong7 I'm still seeing all these dependencies that seem unnecessary, did you look into that?

Reply:
Interesting...I did include the update in git commit and File changes doesn't have this line on me end. Push another commit to see whether it got updated!

Reply:
example-utils is gone, but the other packages I mentioned above are still listed in devDependencies.

Reply:
Ah. I missed them! Removed!